Assets/LeapMotion/Scripts/LeapServiceProvider.cs

```diff
@@ -305,5 +351,62 @@ public class LeapServiceProvider : LeapProvider {
 
       dest.CopyFrom(source).Transform(leapTransform);
     }
+
+    //Precull-Latching Functions
+    protected virtual void OnEnable() {
```

Comment:
OnDisable and OnEnable should be placed up next to all the other Unity event callbacks.